Makefile

```diff
@@ -1,7 +1,8 @@
-RELEASE_DIR = /build/release
+PREFIX:=$(PWD)
+RELEASE_DIR = $(PREFIX)/release
 
-BUILDROOT=/build/buildroot
-BUILDROOT_EXTERNAL=/build/buildroot-external
+BUILDROOT=$(PREFIX)/buildroot
```

Comment:
`PREFIX` is used for target installation prefix in automake world, can we use something else, e.g. `BUILDDIR`?

Reply:
Sure, good point.  I'll make this change, re-check it and push it again.